include/ttmlir/Dialect/TTNN/IR/TTNNOps.td

```diff
@@ -35,6 +37,28 @@ def TTNN_ToMemoryConfigOp : TTNN_Op<"to_memory_config"> {
     let hasVerifier = 1;
 }
 
+def TTNN_ToLayoutOp : TTNN_Op<"to_layout"> {
+    let summary = "ToLayout op.";
+    let description = [{
+    }];
+
+    let arguments = (ins AnyRankedTensor:$input,
+                         TTNN_LayoutAttr:$layout,
+                         TT_Device:$device);
```

Comment:
Not sure if we should change this or not, but all of our other ops take `Value`s first followed by `Attr`s.

Reply:
Good catch, thanks!